### PR TITLE
[SNAP-1308] Dunit test for SNAP-1308 to verify stats of column table in case of bulk insert from SplitMode

### PR DESCRIPTION
**Pull request by SachinJanani**
## Changes proposed in this pull request
* [SNAP-1308] Dunit test for SNAP-1308 to verify stats of column table in case of bulk insert from SplitMode

## Patch testing
Test pass

## ReleaseNotes.txt changes

(Does this change require an entry in ReleaseNotes.txt? If yes, has it been added to it?)

## Other PRs 
https://github.com/SnappyDataInc/snappy-store/pull/139

### CODE REVIEW

**Comment by kneeraj** [comment]
Looks good. It is always nice to add one test with persistence and/or overflow.
Please make a note to add later. Can go ahead with merge now and cherry pick in the 0.71 branch.

File: core/src/main/scala/io/snappydata/SnappyTableStatsProviderService.scala [hunk 1]
```diff
@@ -210,10 +210,14 @@ object SnappyTableStatsProviderService extends Logging {
...
             val itr = pr.localEntriesIterator(null.asInstanceOf[InternalRegionFunctionContext],
               true, false, true, null).asInstanceOf[PartitionedRegion#PRLocalScanIterator]
+            //Resetting PR Numrows in cached batch as this will be calculated every time.
+            //TODO: Decrement count using deleted rows bitset in case of deletes in columntable
+            var rowsInCachedBatch:Long = 0
             while (itr.hasNext) {
-              pr.getPrStats.incPRNumRowsInCachedBatches(itr.next().asInstanceOf[RowLocation]
-                  .getRow(container).getColumn(colPos).getInt)
+              rowsInCachedBatch = rowsInCachedBatch + itr.next().asInstanceOf[RowLocation]
+                  .getRow(container).getColumn(colPos).getInt
```
Comment (sumwale) [inline core/src/main/scala/io/snappydata/SnappyTableStatsProviderService.scala]: @SachinJanani 
More efficient will be:
```
rowsInCachedBatch += itr.next().asInstanceOf[RowLocation].getRow(container)
    .asInstanceOf[AbstractCompactExecRow].getAsInt(colPos, null)
```

Avoids creation of intermediate DataValueDescriptor.
